apps/website/blog/2024-10-29-q3-review.md

```diff
@@ -0,0 +1,61 @@
+
+In terms of current features, the platform supports these gating mechanisms:
+
+- [Semaphore](https://semaphore.pse.dev/)
+- [Hats Protocol](https://www.hatsprotocol.xyz/)
+- Free for all (ie, anyone can join)
+- [EAS](https://docs.attest.org/docs/)
+- Merkle proof
+- [Zupass](https://zupass.org/)
+
+It now also possible to run quadratic and non quadratic voting rounds, as well as run sequential rounds on the same interface. Soon you will also be able to run multiple concurrent rounds.
+
+With the MACI Platform, we were able to support three community rounds, [EthMexico](https://ethmexico.org/), [EthTegucigalpa](https://www.meetup.com/ethereum-tegucigalpa/), and [Cryptoversidad](https://funding.cryptoversidad.com/). All rounds completed successfully with nearly $30,000.00 distributed to several projects and individuals across the three communities. We also received a lot of great feedback that helped improve the platform overall. We couldn’t be happier to work with such wonderful collaborators and help them make a real world impact.
+
+As always, if you're interested in running a round for your community, please get in [touch](https://discord.gg/akDcCB69)
+
+## Coordinator Service ✅
+
+The Coordinator Service is the ultimate automation of MACI’s main pain-points. It now includes three new features for the frontend interface to provide a very neat experience to MACI operators.
+
+Because we prioritise robust security features whenever possible, the service does not hold any wallet seed phrase to communicate with the various chains. Instead, we implemented a flow that uses ephemeral session keys for short lived approvals, greatly decreasing the impact of a key compromise. This is all possible thanks to account abstraction features such as session keys.
+
+With the flow described above, we implemented a deployer service to easily deploy all needed MACI contracts, as well as a an endpoint to merge the MACI state and message trees, an important step required to be able to tally the results.
+
+## Account Abstraction Exploration ✅
+
+Together with a fellow PSE team developer, [John](https://github.com/JohnGuilding), the MACI team is working on a [fork of the MACI platform](https://github.com/privacy-scaling-explorations/hacking-pse) that implements account abstraction features, such as smart wallets and paymasters.
+
+[This fork](https://github.com/privacy-scaling-explorations/hacking-pse) was used in PSE’s first internal hackathon, and allows PSE and EF team members to vote on their favourite hackathon project without spending a single cent from their wallet onchain.
+
+This was a successful experiment which will prove fundamental in our future efforts to improve the MACI platform and make it even more accessible to a general “non crypto native” user base.
+
+Stay tuned for a case study of this experiment!
+
+## MACI Protocol Improvements ✅
```

Comment:
I think here we can add that we also improved integration with maci protocol. There were several tasks that allows to use maci-contracts deployment tasks and override them and some common fixes that allow to integrate easier.